flexget/plugins/sites/__init__.py

```diff
@@ -0,0 +1,36 @@
+"""
+Utilities for search plugins, please try to avoid adding new ones.
+"""
```

Comment:
adding code to __init__.py? not super recommended

Reply:
still better than plugin code in core